notebooks/Exhibit_13_sd(max).ipynb

```
@@ -0,0 +1,120 @@
+   "execution_count": null,
+   "id": "6ec43b5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import math\n",
+    "\n",
+    "import functions  # <- required local module\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from scipy.stats import norm\n",
+    "\n",
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)